src/common/components/list/list.template.njk

```diff
@@ -0,0 +1,31 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% if params.fields and params.payload %}
+  {% set rows = [] %}
+  {% for field in params.fields %}
+    {# Extract the path after $.payload.answers. #}
+    {% set fieldPath = field.ref | replace("$.payload.answers.", "") %}
```

Comment:
Maybe the list should only take key/value pairs and not know the structure of the payload? Move mapping into view model so its easier to test and makes the list component more generic/re-usable?